packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -30,6 +31,12 @@ export abstract class TableColumn<
      */
     public abstract readonly cellRecordFieldNames: readonly TableFieldName[];
 
+    public constructor() {
+        super();
+
+        this.setAttribute('slot', uniqueId('table-column'));
```

Comment:
nit: give a bit more context to the uniqueId in-case we end up with others?
```suggestion
        this.setAttribute('slot', uniqueId('table-column-slot'));
```